src/NHSD.BuyingCatalogue.API/appsettings.json

```diff
@@ -1,14 +1,15 @@
 {
- "AllowedHosts": "*",
+  "AllowedHosts": "*",
 
   "ConnectionStrings": {
-    "BuyingCatalogue": "<YOUR CONNECTION STRING>"
+    "BuyingCatalogue": "Data Source=127.0.0.1,1430;Initial Catalog=buyingcataloguegpit;MultipleActiveResultSets=True;User Id=NHSD;Password=DisruptTheMarket1!"
```

Comment:
Another option here is to manage credentials using the user secrets file.

For reference:
https://docs.microsoft.com/en-us/aspnet/core/security/app-secrets?view=aspnetcore-3.1&tabs=windows

Reply:
Yep, aware but the credentials in this case aren't really a secret (they're available elsewhere in the repo) and having the connection string committed means there's less to do prior to debugging locally.

Reply:
I like this option as it gets other developers familiar with the user secrets feature & hopefully won't lead to sensitive credentials entering source code by always using it. As said in this case the credentials are stated elsewhere in the repo & are just used for local debugging & testing.